integration

```diff
@@ -540,6 +540,7 @@
   "Limych/ha-narodmon",
   "Limych/ha-snowtire",
   "Limych/ha-temperature-feels-like",
+  "Limych/ha-tor_check",
```

Comment:
This seems to be abandoned already?
No movement since this PR was created, and several PRs waiting.

Reply:
All of that PRs are from dependabot — integration is complete.